RxTests/RxCocoaTests/Control+RxTests.swift

```diff
@@ -77,4 +77,24 @@ class ControlTests : RxTest {
         XCTAssertTrue(deallocated)
         XCTAssertTrue(completed)
     }
+
+    func ensureControlObserverHasWeakReference<C, T where C: NSObject>(@autoclosure createControl: () -> (C), _ observerSelector: C -> AnyObserver<T>, _ observableSelector: () -> (Observable<T>)) {
+        var deallocated = false
+
+        autoreleasepool {
+            let control = createControl()
+            let propertyObserver = observerSelector(control)
+            let observable = observableSelector()
+
+            let disposeBag = DisposeBag()
```

Comment:
Hey, looks great.

It would be great if you could move that `disposeBag` above `autoreleasepool`, because in that way we would be better testing is that control being deallocated in spite of subscription lasting.

I've just added similar improvement to control event testing code.

If you could do that and resolve merging conflicts, that would be :100: :)